src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -223,8 +224,8 @@ void func_80A6DE38(EnHorseZelda* this, PlayState* play) {
     checkPos.x = (Math_SinS(this->actor.shape.rot.y) * 30.0f) + this->actor.world.pos.x;
     checkPos.y = this->actor.world.pos.y + 60.0f;
     checkPos.z = (Math_CosS(this->actor.shape.rot.y) * 30.0f) + this->actor.world.pos.z;
-    this->unk_1F4 = BgCheck_EntityRaycastDown3(&play->colCtx, &poly, &bgId, &checkPos);
-    this->actor.shape.rot.x = RAD_TO_BINANG(Math_FAtan2F(this->actor.world.pos.y - this->unk_1F4, 30.0f));
+    this->floorY = BgCheck_EntityRaycastDown3(&play->colCtx, &poly, &bgId, &checkPos);
```

Comment:
floorY is the floor Y forwards, not at the actor's position, so I'd name it like that (maybe `floorYForwards` or InFront :shrug: )
it would also help distinguishing it from `Actor.floorHeight`